Security/Http/Firewall/LegacyTwoFactorListener.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Scheb\TwoFactorBundle\Security\Http\Firewall;
+
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+
+class LegacyTwoFactorListener extends TwoFactorListener
+{
+    public function handle(GetResponseEvent $event)
+    {
+        @trigger_error(sprintf('The "%s()" method is deprecated since Symfony 4.3, use __invoke() instead.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
I'd say we don't need that at all, because the bundle automatically chooses the right solution based on your environment, so no need to tell the user about upgrading something that they don't need to upgrade.